### PR TITLE
Show current unit in classroom sections

### PR DESCRIPTION
**Pull request by larrypo**
Task: Make the **Course** column in this sections-as-student table on the dashboard home page...

image

...able to display assigned course _and_ script information, the same way the teacher's sections table does:

image

### CODE REVIEW

**Comment by islemaster** [comment]
Asking Dave to also take a look at this, since he has recent familiarity with our Course/Unit assignments.

**Comment by davidsbailey** [comment]
👀 looking...

File: apps/test/unit/templates/studioHomepages/SectionsAsStudentTableTest.js [hunk 1]
```diff
@@ -108,13 +108,29 @@ describe('SectionsAsStudentTable', () => {
...
         </td>
       );
-      expect(wrapper).to.containMatchingElement(
-        <td>
-          <a>
-            {section.assignedTitle}
-          </a>
-        </td>
-      );
+      if (section.currentUnitTitle) {
```
Comment (davidsbailey) [inline apps/test/unit/templates/studioHomepages/SectionsAsStudentTableTest.js:112]: It's hard to tell that both parts of this conditional are getting exercised. How about adding a quick check outside of this loop: `expect(wrapper).to.containMatchingElement(<div>Current unit:</div>)`?

**Comment by islemaster** [comment]
[New circle build started](https://circleci.com/gh/code-dot-org/code-dot-org/76704).

**Comment by islemaster** [comment]
Good catch!  I'll merge once Circle passes.  Thanks @larrypo!

**Pull request merged**